### DISCUSSION TITLE
🔗 Include/Exclude taxonomies from the Taxonomy Filter (Duplicate of #23185)

### DISCUSSION THREAD

**Discussion by yodyk** [elementor/elementor#24631] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I have several categories that only apply to specific "Post Types" and would love to be able to manually hide/show specific categories that are relevant to that specific loop grid. 

For example: I am building a Conference Website, within the Agenda I have categories for "Event Day (ex: Day One...)", "Education Track", "Session Type" and "Year (for backend filtering)" with respective child categories within each. 

The agenda is already broken out by Day via tabs because it creates a more user-friendly experience than having a full layout of 3 days of agenda sessions. Therefor, there would be no need to have "Event Day" Category/Child Categories show, where as it would be great for the users to be able to filter by Education Track and Type of Session. (For example, they're specifically looking for "Marketing + Workshop" session.

### Describe the solution you'd like.

It would be great if I could Hide/Show certain categories that apply to certain post types within the Query Filter via Manual Selection Include/Exclude. As the Loop Grids are able to do. 

### Describe alternatives you've considered.

It's possible to do it via separate loops grids placed inside a second nested tab but that creates layers of Tabs which is not a good look on the user-experience side of things and doesn't create the ability to "multi-filter" like my example above of a user wanting to find a specific "Marketing Track + Workshop" Session

Tab 1: Event Day Tab
    Tab 2: Education Track Tab / Session Type Tab 

### Additional context

Here is the conference agenda for reference: 
https://digitalmarketingstrategies.org/agenda/

I am utilizing these plugins: ACF Pro for custom fields & Custom Post Types UI for the Seperate types of Posts (Speakers, Sponsors, Sessions, etc.)

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there, @yodyk!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #23185 

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
